Source/UINavigation/Private/UINavWidget.cpp

```diff
@@ -1496,7 +1496,7 @@ void UUINavWidget::OnPreSelect(int Index)
 		SetKeyboardFocus();
 		bWaitForInput = true;
 	}
-	else
+	else if (CurrentButton != nullptr)
```

Comment:
Ideally you'd check if the CurrentButton is null at the start of this function, and if it is, return.